Configuration/TCA/Overrides/tt_content.php

```diff
@@ -18,6 +19,13 @@ static function (): void {
             'EXT:tea/Resources/Public/Icons/Extension.svg'
         );
 
+        // These two commands add the flexform configuration for the plugin.
+        $GLOBALS['TCA']['tt_content']['types']['list']['subtypes_addlist']['tea_teaindex'] = 'pi_flexform';
+        ExtensionManagementUtility::addPiFlexFormValue(
+            'tea_teaindex',
```

Comment:
The string `tea_teaindex` should be our plugin signature.
This should be available as return value of above `registerPlugin()` call.
I guess it would make sense to refactor the lines to store the return value, e.g. in `$pluginSignature` and use it here:

```suggestion
        $GLOBALS['TCA']['tt_content']['types']['list']['subtypes_addlist'][$pluginSignature] = 'pi_flexform';
        ExtensionManagementUtility::addPiFlexFormValue(
            $pluginSignature,
```

This is a new feature since 12.0, and therefore available in our code base: https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/12.0/Feature-82809-MakeExtensionUtilityregisterPluginMethodReturnPluginSignature.html

Reply:
Done.